templates/contactList.html

```diff
@@ -13,4 +13,7 @@
 			<h2>{{ctrl.t.emptySearch}}</h2>
 		</div>
 	</div>
+	<div ng-if="filtered.length>0" class="contacts-list-count">
+		{{ filtered.length }} {{ filtered.length==1 ? ctrl.t.oneContact : ctrl.t.multipleContacts }}
```

Comment:
Yes, will have a look. If anyone has a hint how to solve the plural translations, that would be great. Or is it basically just moving to translation strings with placeholders like "%d contact"/"%d contacts"